mass_mailing_gdpr_limitations/__manifest__.py

```diff
@@ -1,12 +1,13 @@
 {
     "name": "AF Mass Mailing gdpr limitations",
     "summary": "This module limits the disclosure of personal data in the mass-mailing module",
-    "version": "12.0.1.1.0",
+    "version": "12.0.1.1.1",
```

Comment:
I would argue that changing the technical name of a module is a backwards incompatible change that affects other modules that might be inheriting this module.
Therefore version should be 12.0.2.0.0